### PR TITLE
Added a line while running build.bat file

### PR DESCRIPTION
**Pull request by i-am-yuvi**
## Describe your changes
Added a condition while running build.bat

## This PR fixes:
#735

* Resolved #

## Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -12,13 +12,20 @@ if exist C:\MSYS64\ (
 ) else (
     mkdir temp_install & cd temp_install
 
+    if exist C:\Windows\System32\curl (
```
Comment (dwertent) [inline build.bat:15]: Why are you looking for the `curl` in a specific apathy and searching it in the `$PATH`?

File: core/mocks/loadmocks.go [hunk 2]
```diff
@@ -4,15 +4,15 @@ import (
 	"encoding/json"
 
 	"github.com/armosec/armoapi-go/armotypes"
-	"github.com/kubescape/k8s-interface/workloadinterface"
-	"github.com/kubescape/opa-utils/reporthandling"
+	"github.com/armosec/k8s-interface/workloadinterface"
```
Comment (dwertent) [inline core/mocks/loadmocks.go:7]: This should not be changed

File: core/pkg/resultshandling/results_test.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
...tyContext.runAsNonRoot == false\n\tpath = sprintf(\"%vcontainers[%v].securityContext.runAsNonRoot\", [begginingOfPath, format_int(i, 10)])\n}\n\nisRootContainer(container, i, begginingOfPath) = path  {\n\tpath = \"\"\n    container.securityContext.runAsUser == 0\n\tpath = sprintf(\"%vcontainers[%v].securityContext.runAsUser\", [begginingOfPath, format_int(i, 10)])\n}\n\nisRootContainer(container, i, begginingOfPath) = path  {\n\tpath = \"\"\n     container.securityContext.runAsGroup == 0\n\t path = sprintf(\"%vcontainers[%v].securityContext.runAsGroup\", [begginingOfPath, format_int(i, 10)])\n}","resourceEnumerator":"","ruleLanguage":"Rego","match":[{"apiGroups":["*"],"apiVersions":["*"],"resources":["Deployment","ReplicaSet","DaemonSet","StatefulSet","Job","Pod","CronJob"]}],"ruleDependencies":[],"configInputs":null,"controlConfigInputs":null,"description":"fails if container can run as root","remediation":"Make sure that the user/group in the securityContext of pod/container is set to an id less than 1000, or the runAsNonRoot flag is set to true. Also make sure that the allowPrivilegeEscalation field is set to false","ruleQuery":"armo_builtins"}],"rulesIDs":[""],"baseScore":6}]}`
```
Comment (dwertent) [inline core/pkg/resultshandling/results_test.go:4]: What changed here?

**Review by dwertent — CHANGES_REQUESTED** [review]

File: build.bat (hunk 1 above)
Comment (i-am-yuvi) [reply build.bat:15]: I was just adding a condition whether curl is present in his path.

**Comment by i-am-yuvi** [comment]
Sorry just did many mistakes

**Pull request closed without merging**